frontend/src/concepts/pipelines/content/compareRuns/CompareRunTableRow.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Td, Tr } from '@patternfly/react-table';
+import { PipelineRunKFv2 } from '~/concepts/pipelines/kfTypes';
+import { CheckboxTd } from '~/components/table';
+import {
+  RunCreated,
+  RunDuration,
+  RunStatus,
+} from '~/concepts/pipelines/content/tables/renderUtils';
+import PipelineRunTableRowTitle from '~/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRowTitle';
+import useExperimentById from '~/concepts/pipelines/apiHooks/useExperimentById';
+import usePipelineRunVersionInfo from '~/concepts/pipelines/content/tables/usePipelineRunVersionInfo';
+import { PipelineVersionLink } from '~/concepts/pipelines/content/PipelineVersionLink';
+
+type CompareRunTableRowProps = {
+  isChecked: boolean;
+  onToggleCheck: () => void;
+  run: PipelineRunKFv2;
+};
+
+const CompareRunTableRow: React.FC<CompareRunTableRowProps> = ({
+  isChecked,
+  onToggleCheck,
+  run,
+}) => {
+  const [experiment] = useExperimentById(run.experiment_id);
+  const { version, loaded: isVersionLoaded, error: versionError } = usePipelineRunVersionInfo(run);
+
+  return (
+    <Tr>
+      <CheckboxTd id={run.run_id} isChecked={isChecked} onToggle={onToggleCheck} />
+      <Td dataLabel="Run">
+        <PipelineRunTableRowTitle run={run} />
```

Comment:
It looks like UX is looking for a slightly different title. Theirs includes a description and no tag for the `one-off` or `scheduled`
<img width="325" alt="Screenshot 2024-04-04 at 9 42 24 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/12587674/3ef6ba11-0058-4753-98c5-45cb426fd797">

Reply:
<img width="1728" alt="Screenshot 2024-04-04 at 3 16 03 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/391d02d6-2b9e-4fa0-9fa9-90b38a234adc">

This is the best output I can get, title needs to be bold IMO. And long string will be truncated with a tooltip.